### PR TITLE
Added Jelastic guide on how to restrict access to nginx server

### PR DESCRIPTION
**Pull request by evgeluev**

### CODE REVIEW

File: content/en/docs/jelastic-paas/guides/nginx_restrict_access.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Add the following to the server section in nginx.conf.
+
+
+```bash
+allow 1.2.3.4;
+allow 5.6.7.8; 
+deny all;
+```
+
+If you don't have a public IP on your nginx then add the following to the http section in nginx.conf.
```
Comment (Doctor-love) [inline content/en/docs/jelastic-paas/guides/nginx_restrict_access.md:23]: "You'll also need to add..." ?

File: content/en/docs/jelastic-paas/guides/nginx_restrict_access.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Nginx access restriction"
```
Comment (Doctor-love) [inline content/en/docs/jelastic-paas/guides/nginx_restrict_access.md:2]: Mayhaps "Nginx IP access restriction"?

File: content/en/docs/jelastic-paas/guides/nginx_restrict_access.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+description: "Restrict access to your nginx server"
+weight: 1
+alwaysopen: true
+---
+
+## Overview
+
+If you want to restrict access to your web site by limiting the source IP addresses that can access the site this is how you do it. Replace the IP addresses with the IP you want to allow access from.
+
+## Edit nginx.conf.
```
Comment (Doctor-love) [inline content/en/docs/jelastic-paas/guides/nginx_restrict_access.md:12]: Something like "Configuring nginx" maybe?

File: content/en/docs/jelastic-paas/guides/nginx_restrict_access.md (hunk 1 above)
Comment (Doctor-love) [reply content/en/docs/jelastic-paas/guides/nginx_restrict_access.md:23]: What we are trying to say is that if they don't have have a public address, they also need to do XYZ.

File: content/en/docs/jelastic-paas/guides/nginx_restrict_access.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+allow 1.2.3.4;
+allow 5.6.7.8; 
+deny all;
+```
+
+If you don't have a public IP on your nginx then add the following to the http section in nginx.conf.
+
+```bash
+real_ip_header X-Forwarded-For; 
+set_real_ip_from 10.0.0.0/8;
```
Comment (Doctor-love) [inline content/en/docs/jelastic-paas/guides/nginx_restrict_access.md:27]: Maybe a note should be added regarding the consequences/risks of doing this? Any tenant could spoof an address in this case.

File: content/en/docs/jelastic-paas/guides/nginx_restrict_access.md (hunk 1 above)
Comment (bl0m1) [reply content/en/docs/jelastic-paas/guides/nginx_restrict_access.md]: I think this section could be improved. This is not based on if you have a public IP or not more if you have another proxy/loadbalancer or waf in front of your instance?

This part also requires you to enable x-forwarded-for in the lb/proxy or waf

**Pull request closed without merging**